src/components/TextFormatter.js

```diff
@@ -0,0 +1,69 @@
+import React, { PureComponent } from 'react';
+//import figlet from 'figlet';
+
+export default class PrintText extends PureComponent {
+  state = {
+    text: '',
+    color: 'black'
+    //    font: 'banner3-D'
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();  
+  }
+
+  // formatText = () => {
+  //   const { text, font } = this.state;
+  //   figlet.text(text, {
+  //     font: font
+  //   }, (err, result) => {
+  //     this.setState({ text: result });
+  //   });
+  // }
+
```

Comment:
Just a FYI, you're able to combine these two handles by writing the function like so...

``` handleChange = ({ target }) => {
              this.setState({ [target.name]: target.value });
       }
```